libs/platform/src/lib/thumbnail/thumbnail-image/thumbnail-image.component.ts

```diff
@@ -95,7 +95,10 @@ export class ThumbnailImageComponent implements OnChanges, OnInit {
     }
 
     /** @hidden */
-    thumbnailClick(selectedMedia: Media): void {
+    thumbnailClick(event: KeyboardEvent, selectedMedia: Media): void {
```

Comment:
you are using this for `click` and `keydown` events, so event is either of type `KeyboardEvent` or `MouseEvent`